RELEASE/scripts/sl_ascend.ash

```diff
@@ -14784,6 +14784,13 @@ void sl_begin()
 
 	dailyEvents();
 	consumeStuff();
+
+	// Prevent accidental drunken stupor
+	if(my_familiar() == $familiar[Stooper])
+	{
+		handleFamiliar($familiar[none]);
```

Comment:
this'll need to actually be a rare instance of `use_familiar` I think, because `handleFamiliar` just sets the familiar to be switched to during presool